app/src/main/java/sopt/org/millie/presentation/search/searchresult/SearchUiState.kt

```diff
@@ -13,6 +13,6 @@ data class SearchUiState(
     val searchTabs: List<String> = SearchResultConstants.SEARCH_TABS,
     val selectedTab: String = SearchResultConstants.SELECTED_TAB,
     val searchBookList: UiState<List<SearchBookModel>> = UiState.Loading,
-    val searchLibraryList: UiState<List<SearchLibraryModel>> = UiState.Loading,
+    val searchLibraryList: List<SearchLibraryModel> = emptyList(),
```

Comment:
```suggestion
    val searchLibraryList: List<SearchLibraryModel> = persistentListOf(),
```
emptyList()보다 persistentListOf()가 더 좋을 것 같아욯ㅎ emptyList()는 가변 리스트로 변환되어 사용될 가능성이 있어 Compose 상태 관리에서 불변성 보장이 약해질 수 있어요 !!! persistentListOf()는 immutable 기반의 불변 리스트라  상태의 Immutable 특성을 명확하게 유지하구 항목 변경 시에도 새로운 인스턴스를 반환하여 Compose의 recomposition 안정성도 더 좋아진답니다 ㅎㅎㅎ

Reply:
넹 알겠습니다!